project_task_dependency/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
```

Comment:
@tarteo IMHO Add website attribute, for reference  [here](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)